storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/KvStoreCombinedDao.java

```diff
@@ -182,6 +182,12 @@ default void addHotStates(final Map<Bytes32, BeaconState> states) {
 
     void addHotStateRoots(Map<Bytes32, SlotAndBlockRoot> stateRootToSlotAndBlockRootMap);
 
+    default void addHotBlobSidecars(final Map<SlotAndBlockRoot, List<BlobSidecar>> blobSidecars) {
+      blobSidecars.forEach(this::addHotBlobSidecarSlot);
+    }
+
+    void addHotBlobSidecarSlot(SlotAndBlockRoot slotAndBlockRoot, List<BlobSidecar> blobSidecars);
```

Comment:
maybe `addHotBlobSidecarForBlock`?

Reply:
let's keep it, as we are going to modify everything on this way